recipes/metacerberus-lite/meta.yaml

```diff
@@ -6,11 +6,11 @@ package:
   version: "{{ version }}"
 
 source:
-  url: "https://files.pythonhosted.org/packages/source/m/MetaCerberus/MetaCerberus-{{ version }}.tar.gz"
-  sha256: 725be35a0e9b2a3867791e1cb8284ec95a09d3479516e9b265e3751659ce7935
+  url: "https://github.com/raw-lab/MetaCerberus/releases/download/v1.3.1/MetaCerberus-1.3.1.post1.tar.gz"
```

Comment:
Switching to the GitHub releases page is good, but this should continue using the {{version}} tag, rather than hard-coding the number. Also, why was there a post1 version with only a tar.gz uploaded and no tagged release? Ideally, you'd just pull the GH autogenerated tar ball. If that's not possible, it should be documented in a comment.

Reply:
Thank you for your feedback.
The intention was just a quick patch, and the reason for the post1 uploaded was because GH adds all the files that are not needed for the release that made the file size larger than it needed to be. I excluded unnecessary files and created a new release as 1.3.2, and have made the update. I hope this is good.
Thank you.